EpicLoot/config/translations.json

```diff
@@ -739,6 +751,18 @@
     "mod_epicloot_me_modifyattackstaminause_prefix1": "Gladiator's",
     "mod_epicloot_me_modifyattackstaminause_suffix1": "the Gladiator",
     "mod_epicloot_me_modifyattackstaminause_desc": "Reduce the player's attack stamina usage by -<b><color=yellow>X</color></b>%.",
+    "mod_epicloot_me_bloodlust_display": "Bloodlust",
+    "mod_epicloot_mebloodlust_prefix1": "Vampires",
```

Comment:
Fix this to match others, should be ``_me_``. Should Vampires have an apostrophe "Vampire's"?

Reply:
Fixed in: https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/cba07184b6ec88bc83a55b32dc6d6500d80e1fb7